sdk/core/core-amqp/test/cbs.spec.ts

```diff
@@ -64,7 +67,7 @@ describe("CbsClient", function () {
     it("honors abortSignal", async function () {
       const connectionStub = new Connection();
       // Stub 'open' because creating a real connection will fail.
-      stub(connectionStub, "open").resolves({} as any);
+      vi.spyOn(connectionStub, "open");
```

Comment:
This allows us to spy on the implementation as well as to mock it.  Yes, we could have easily done vi.fn().mockImplementation instead.  This is easier in this case.

Reply:
Yeah, I definitely support using spies instead of mocks / stubs when we can, it's just the comment above threw me off - maybe it's outdated as comments sometimes are? `// Stub 'open' because creating a real connection will fail.`

Either way not a big deal if the test passes!

Reply:
Yep, tests pass for now and we can revisit later.